tests/realtikvtest/scripts/bootstrap-common.sh

```
@@ -0,0 +1,96 @@
+    else
+        killall -9 -r -q tikv-server || true
+    fi
+}
+
+function start_tikv_cluster() {
+    local config_dir="$1"
+    local data_base_dir="$2"
+
+    for attempt in {1..3}; do
+        echo "Starting TiKV cluster (attempt ${attempt}/3)"
+        bin/tikv-server --config=${config_dir}/tikv.toml --data-dir=${data_base_dir}/tikv-0/data --addr=127.0.0.1:20160 --advertise-addr=127.0.0.1:20160 --status-addr=127.0.0.1:20180 --pd=${pd_client_urls} --log-file=tikv0.log &
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)